packages/infra/lib/api-stack/ccda-search-connector.ts

```diff
@@ -129,6 +129,8 @@ export function setup({
     lambdaLayers: [lambdaLayers.shared],
     envType,
     alarmSnsAction,
+    alarmMaxAgeOfOldestMessage: Duration.minutes(1),
+    alarmMaxAgeOfOldestMessageDlq: Duration.minutes(5),
```

Comment:
OOC - why isn't this just 1m? Don't we want to react to DLQ messages right away?

Reply:
We have an alarm for when the DLQ has more than one item, this is just an additional alarm if we don't act on that one after 5min.